flocker/node/_deploy.py

```diff
@@ -420,8 +420,8 @@ def applications_from_units(result):
                 running=running,
                 not_running=not_running,
                 used_ports=self.network.enumerate_used_ports(),
-                # FLOC-1181 will want to add here the manifestations that
-                # are not attached to any application.
+                # FLOC-1181 will want to add here the remaining
+                # manifestations that are not attached to any application.
```

Comment:
That is to say, all the manifestations that the above loop didn't consume from `available_manifestations`?